crates/red_knot_python_semantic/resources/mdtest/scopes/moduletype_attrs.md

```diff
@@ -56,10 +56,10 @@ inside the module:
 import typing
 
 reveal_type(typing.__name__)  # revealed: str
-reveal_type(typing.__init__)  # revealed: Literal[__init__]
+reveal_type(typing.__init__)  # revealed: @Todo(bound method)
 
 # These come from `builtins.object`, not `types.ModuleType`:
-reveal_type(typing.__eq__)  # revealed: Literal[__eq__]
+reveal_type(typing.__eq__)  # revealed: @Todo(bound method)
```

Comment:
these look like regressions but they are actually improvements. accessing `typing.__init__` does not actually return the original `__init__` function in `types.ModuleType`'s class body; it returns a bound method. The difference is that you don't need to pass `self` in explicitly when calling the method.